tx-source/site_core.php

```diff
@@ -51,6 +51,7 @@
 $Definition['%s hour'] = '%s hour';
 $Definition['%s hours'] = '%s hours';
 $Definition['%s Insightfuls'] = '%s Insightfuls';
+$Definition['%s is'] = '%s is';
```

Comment:
This is probably not safe to translate this way. Other language cannot create sentences the same way as English due to masculine/feminine words and verb conjugation.